srcpkgs/flake8/template

```diff
@@ -1,22 +1,26 @@
 # Template file for 'flake8'
 pkgname=flake8
-version=3.8.4
+version=3.9.2
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-pycodestyle>=2.6.0<2.7.0
- python3-pyflakes>=2.2.0<2.3.0 python3-mccabe>=0.6.0<0.7.0
- python3-entrypoints>=0.3<0.4"
-checkdepends="python3-mock python3-pycodestyle python3-pytest python3-pyflakes"
+depends="python3-pycodestyle>=2.7.0<2.8.0 python3-pyflakes>=2.3.0<2.4.0
+ python3-mccabe>=0.6.0<0.7.0 python3-entrypoints>=0.3<0.4"
+checkdepends="$depends python3-mock python3-pytest"
```

Comment:
I don't like this super restrictive version bounds, I think we should remove them. They are hard to maintain and parse.

Reply:
https://flake8.pycqa.org/en/latest/faq.html#why-does-flake8-use-ranges-for-its-dependencies
> Flake8 uses ranges for mccabe, pyflakes, and pycodestyle because each of those projects tend to add new checks in minor releases. It has been an implicit design goal of Flake8’s to make the list of error codes stable in its own minor releases. That way if you install something from the 2.5 series today, you will not find new checks in the same series in a month from now when you install it again.

It seems like the version restrictions are not really necessary on void, I removed them.